index.js

```diff
@@ -1,6 +1,16 @@
 const WHITE = [255, 255, 255, 1]
 const BLACK = [0, 0, 0, 1]
 
+// Cache for CSS color values to avoid repeated getComputedStyle calls
+let colorCache = new Map()
+
+/**
+ * Clear the color cache
+ */
+const clearColorCache = () => {
```

Comment:
@JohnHuffman824 in what circumstances will this be used? If we can't think of any, we should follow YAGNI and remove the method.